gitinstall-hp.md

```diff
@@ -0,0 +1,62 @@
+# Installing git on HP-UX
+
+## Using Depot packages
```

Comment:
Please add a table of contents section. See other pages for installing git to see how I did this.